src/main/scala/weld/WeldContext.scala

```diff
@@ -0,0 +1,49 @@
+package weld
+
+class WeldContext private(handle: Long) extends WeldManaged(handle) {
+  override protected def doClose(): Unit = {
+    WeldJNI.weld_context_free(handle)
+  }
+
+  /**
+   * Obtain the memory usage of this context.
+   */
+  def memory_usage(): Long = {
+    checkAccess()
+    WeldJNI.weld_context_memory_usage(handle)
+  }
+
+  /**
+   * Create a cleaner for the managed object.
+   */
+  override protected def cleaner = new WeldContext.Cleaner(handle)
+}
+
+object WeldContext {
+  /**
+   * Initialize a [[WeldContext]] using the default configuration.
+   */
+  def init(): WeldContext = {
+    val conf = new WeldConf
+    try init(conf) finally {
+      conf.close()
```

Comment:
Why close the conf?

Reply:
This method is just initializing a default context (similar to what's happening in WeldModule.scala) -- the `init` below supports passing in a `conf`.

Reply:
Got it I misunderstood. I thought the conf was held onto within the context, but we just parse out some info from the conf and then it's safe to throw away.